.vimrc.after

```diff
@@ -0,0 +1,28 @@
+colorscheme solarized 
```

Comment:
file `.vimrc.after` is intended for local overriders, not for the generic `dotvim` repository